packages/cli/src/bin.ts

```diff
@@ -57,7 +57,7 @@ process.removeAllListeners('warning')
 
 // Listen to Ctr + C and exit
 process.once('SIGINT', () => {
-  process.exit(130)
+  process.exitCode = 130
```

Comment:
This will prevent the CLI from exiting at all when _not_ using subcommands with their own sigint handlers, at least when running it directly and not in npm script. Should we maybe check if there are any other handlers, and exit if there are none?

```suggestion
  if (process.listenerCount('SIGINT') > 0) {
    process.exitCode = 130
  } else {
    process.exit(130)
  }
```

(comparing to 0 and not to 1 because this event handler is attached with `once` and not `on`, and the count will have already been decremented by the time the callback is invoked)

Reply:
Great idea to preserve original behavior for downstream that doesn't listen as well!

I have pushed a slightly different variation to what you suggested. 
`process.exit()` exits with `process.exitCode` if it was set before.